lib/crack.rb

```diff
@@ -0,0 +1,38 @@
+require_relative './cryptor'
+require_relative './decryptor'
+
+
+class Crack < Cryptor
+
+  def initialize(message, date = Time.now.strftime('%d%m%y')) 
+    @message = message 
+    @date = date
+  end
+  
+  def make_key_string(number)
```

Comment:
generates a `key` string by padding a number between 0 and 99999 with zeroes such that the resulting string has five characters